spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/TagsConfigurationProperties.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.springboot;
+
+import org.axonframework.config.TagsConfiguration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Spring Configuration properties for {@link TagsConfiguration}.
+ *
+ * @author Milan Savic
+ * @since 4.2
+ */
+@ConfigurationProperties(prefix = "axon")
+public class TagsConfigurationProperties {
+
+    /**
+     * Tags represented by key-value pairs.
+     */
+    private Map<String, String> tags = new HashMap<>();
+
+    /**
+     * Gets tags.
```

Comment:
Same super-duper-nitty issue as I placed on the `TagsConfiguration`. I'd expect getters/setters and `@param` fields to specify the type being returned/set/provided.